docs/upgrade_tails_usbs.rst

```diff
@@ -0,0 +1,124 @@
+Updating Tails USBs
+===================
+
+There are 3 components of SecureDrop that use Tails USBs:
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
+-----------------
+
+For the Internet-connected Tails workstations (the *Admin Workstation* and
+the *Journalist Workstation*), you'll see a notification when a new version
+of Tails is available. We recommend that you :ref:`back up your existing
+configuration<Step 1>` and upgrade as soon as possible.
+
+|Upgrade Notification|
+
+It usually takes some time for updates to download, so keep that in mind when choosing when to upgrade.
+
+.. |Upgrade Notification| image:: images/tails_upgrade_notification.png
+
+
+Manual Updates
+--------------
+
+However, because the *Secure Viewing Station (SVS)* is airgapped, it cannot
+receive upgrade notifications, so it will need to be updated manually.
+
+.. important:: 
+        The *SVS* stores some of SecureDrop's most sensitive data (decrypted submissions, the SecureDrop Application private key), so we **strongly** encourage you to manually upgrade the *SVS* whenever a new version of Tails is released.
+
+The Tails project releases updates `every 6 weeks`_. Occasionally they
+release a new version ahead of the normal cycle in order to address a
+security issue. For regular Tails OS and security information, check out the
+Tails `Security page`_ and subscribe to the Tails RSS/Atom feed.
+
+.. _every 6 weeks: https://tails.boum.org/support/faq/index.en.html
+.. _Security page: https://tails.boum.org/security/index.en.html
+
+
+Tails Manual Update Process at a Glance
+----------------------------------------
+
+The process for manually updating a Tails workstation is:
+
+  #. Recommended: :ref:`Make a backup of the Tails workstation USB that you intend to upgrade<Step 1>`. For simplicity, we'll refer to the workstation you want to upgrade as the *Secure Viewing Station*, although this process would work on any Tails USB.
+  #. Install the :ref:`latest version of Tails<Step 2>` on your *primary Tails USB*.
+  #. Use the primary Tails USB to :ref:`perform a manual update of the Secure Viewing Station<Step 3>` USB on an offline (airgapped) machine.
```

Comment:
Instead of "an offline (airgapped) machine", I would suggest "on the airgapped computer serving as the SVS", to map it more closely against the detailed instructions later.